src/main/java/lz/modelTesting/controllers/HighlightsController.java

```diff
@@ -0,0 +1,69 @@
+package lz.modelTesting.controllers;
+
+
+
+import lz.modelTesting.documents.Highlight;
+import lz.modelTesting.repositories.HighlightsRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.List;
+
+@RestController
+@RequestMapping("/highlights")
+@CrossOrigin(origins = "*", allowedHeaders = "*")
+public class HighlightsController {
+
+    private final static String MOCKUP_ID = "mockupId";
```

Comment:
that could be reused from `SurveysController` and even better extracted to another place (another class) and be used here and in `SurveysController` from that another place